src/config.xml

```diff
@@ -3802,6 +3802,130 @@ where `loc1` and `loc2` can be relative or absolute paths or URLs.
 ]]>
       </docs>
     </option>
+    <option type='list' id='DOT_GRAPH_ATTR' format='string' depends='HAVE_DOT'
+      defval='bgcolor=\"transparent\"'>
```

Comment:
I don't think the double quotes should be escaped (`\"`), it is not necssary as the string is embedded in single quotes , probably the double quotes can removed all together (see e.g. `DOT_COMMON_ATTR`)

Reply:
Escaping the double quotes is necessary to create the same output as before:
```
  bgcolor="transparent";
```

I think having un-escaped `"` would confuse the parsing of the doxyfile?

You are correct that dot also accepts `bgcolor=transparent`, but I was trying to keep the output the same.